src/python/pants/core/goals/run.py

```diff
@@ -53,7 +53,7 @@ class RunInSandboxBehavior(Enum):
     The behaviors are as follows:
 
     * `RUN_REQUEST_HERMETIC`: Use the existing `RunRequest`-generating rule, and enable cacheing.
-       Use this if you are confident the behaviour of the rule relies only on state that is
```

Comment:
As a British person I will wave the flag for accepting British spelling, but again no need to revert this.

Reply:
In this case, the object itself is called "Behavior" so I think the doc needs to consistently use the same spelling.

Reply:
I'm used to British spelling, but will accept American spelling as well--the most important is to be consistent, imho.